egs/multi_zh-hans/ASR/local/train_bpe_model.py

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env python3
+# Copyright    2021  Xiaomi Corp.        (authors: Fangjun Kuang)
```

Comment:
Please replace it with your name.

By the way, can we replace this file with a symlink?

Reply:
This one has a dew differences comparing with the original version (e.g. token_coverage, byte_fallback ...), thus cannot be replaced by a symlink

Reply:
> This one has a dew differences comparing with the original version (e.g. token_coverage, byte_fallback ...), thus cannot be replaced by a symlink

Please add a comment describing the difference.